Formula/a/apigeecli.rb

```diff
@@ -0,0 +1,37 @@
+class Apigeecli < Formula
+  desc "Apigee management API command-line interface"
+  homepage "https://cloud.google.com/apigee/docs"
+  url "https://github.com/apigee/apigeecli/archive/refs/tags/v2.13.0.tar.gz"
+  sha256 "61d441d082d07dd7102483dea826fa57dbd97d76434076312a67b067776698fa"
+  license "Apache-2.0"
+  head "https://github.com/apigee/apigeecli.git", branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_releases
```

Comment:
I think this is fine or remove livecheck to get recent tags.
```suggestion
    strategy :github_latest
```